packages/terra-clinical-result/src/terra-dev-site/doc/ClinicalResult/AccessibilityGuide.6.doc.mdx

```diff
@@ -0,0 +1,199 @@
+    import ThemeContext from 'terra-theme-context';
+    import styles from '../Examples.module.scss';
+
+    const cx = classNamesBind.bind(styles);
+
+    const ResultNameHeaderCellAccessibleNamesExample = () => {
+      const theme = React.useContext(ThemeContext);
+      const mockFlowsheetClassnames = classNames(
+        cx(
+          'mock-flowsheet-example',
+          theme.className,
+        ),
+      );
+
+      return (
+        <div className={mockFlowsheetClassnames}>
+          <div className={cx('mock-flowsheet-firstcolumn')}>
+            <div className={cx('mock-flowsheet-firstcolumn-cell')}>
+              <ResultNameHeaderCell resultName="Temp" fullResultName="Temperature" unit="degC" fullUnit="Degrees Celsius" />
+            </div>
+          </div>
+        </div>
+      );
+    };
+
+    export default ResultNameHeaderCellAccessibleNamesExample;
+
+</div>
+
+This example shows how to set up the optional `fullResultName` and `fullUnit` props in order to provide accessible names for the displayed result name and unit.
+
+</Notice>
+
+### Other Considerations
+
+#### Flowsheet Result Cell with Multiple Results
+
+Flowsheet Result Cell is used for the grid display of a result collection and can handle displaying an indication of multiple results. When there are multiple results, the first result is displayed along with a custom symbol that is used to communicate information about the remaining results:
+
+<FlowsheetResultCellMultipleResultsSymbols />
+
+The number represents how many additional results exist and the color of the result stack indicates the severity of the **most** serious result in the side panel.
+
+Alternative text has been added to communicate the number of additional results and the severity:
+
+| Severity Color | Meaning | Text to be Read |
+| --- | --- | --- |
+| Gray | Normal | "A total of [number] results are available." |
+| Light Red | Out of range (high or low) but not critical | "A total of [number] results are available with at least 1 out of range result." |
+| Dark Red | Critical | "A total of [number] results are available with at least 1 critical result." |
+
+e.g. if we have a dark red box with 3 results the screenreader will read out:
+
+“A total of 3 results are available with at least 1 critical result.”
+
+#### Usage of Visually Hidden Text
+
+There are a few cases in which we leverage visually hidden text to give greater context to certain styling on the front end - or related to statuses of certain results. These cases are as follows:
+
+##### No Results:
+
+When the `hasResultNoData` prop is used the display will show: '--'
+
+Hidden text has been added to read out "No Results" when the `hasResultNoData` prop is used.
+
+##### Entered In Error:
+
+When the `hasResultError` prop is used the display will show: <ClinicalResultKnownError style={{display: 'inline'}}/>
+
+Hidden text has been added to read out "Value Uncharted as In Error" to convey this has a status of `entered-in-error`.
```

Comment:
"Entered In Error" is specifically used in [FlowsheetResultCell](https://engineering.cerner.com/terra-ui/components/terra-clinical-result/clinical-result/flowsheet-result-cell) and it shows like this:
<img width="559" alt="Screenshot 2023-07-11 at 9 13 59 AM" src="https://github.com/cerner/terra-clinical/assets/44185348/509868d7-e669-4c77-b000-c4c0034c6ef1">
I think the KnownError is another error

Reply:
Good catch, fixed here: https://github.com/cerner/terra-clinical/pull/884/commits/98d4b581f2aef37be6f5b8152582b13f036d38ea